test/sqllogictest/transform/predicate_pushdown.slt

```diff
@@ -326,37 +334,17 @@ EXPLAIN PHYSICAL PLAN AS TEXT FOR
 SELECT * FROM t1, t2 WHERE t1.f1 = t2.f1 AND ((t1.f2 = 3 AND t2.f2 = 4) OR (t1.f2 = 5 AND t2.f2 = 6));
 ----
 Explained Query:
-  Join::Delta
-    plan_path[0]
-      final_closure
-        project=(#0, #1, #0, #2)
-      delta_stage[0]
-        closure
-          project=(#0, #1, #4)
-          filter=((#5 OR #6) AND ((#2 AND #5) OR (#3 AND #6)))
-          map=((#4 = 4), (#4 = 6))
-        lookup={ relation=1, key=[#0] }
-        stream={ key=[#0], thinning=(#1..=#3) }
-      initial_closure
-        project=(#0..=#3)
-        filter=((#0) IS NOT NULL AND (#2 OR #3))
-        map=((#1 = 3), (#1 = 5))
-      source={ relation=0, key=[#0] }
-    plan_path[1]
-      final_closure
-        project=(#0, #1, #0, #2)
-      delta_stage[0]
-        closure
-          project=(#0, #4, #1)
-          filter=((#0) IS NOT NULL AND (#5 OR #6) AND ((#5 AND #2) OR (#6 AND #3)))
-          map=((#4 = 3), (#4 = 5))
-        lookup={ relation=0, key=[#0] }
-        stream={ key=[#0], thinning=(#1..=#3) }
-      initial_closure
-        project=(#0..=#3)
-        filter=((#2 OR #3))
-        map=((#1 = 4), (#1 = 6))
-      source={ relation=1, key=[#0] }
+  Join::Linear
```

Comment:
Very large change here worth investigating.

Reply:
Nope. Just a change from delta to linear join, and a large amount of description of the join that changes.